lib/bosh/gen/generators/new_release_generator/templates/templates/make_manifest.tt

```diff
@@ -59,7 +59,7 @@ perl -pi -e "s/PLACEHOLDER-DIRECTOR-UUID/$DIRECTOR_UUID/g" $tmpdir/stub-with-uui
 perl -pi -e "s/NAME/$NAME/g" $tmpdir/stub-with-uuid.yml
 perl -pi -e "s/STEMCELL/$STEMCELL/g" $tmpdir/stub-with-uuid.yml
 
-spiff merge \
+spruce merge --prune meta \
```

Comment:
Can you do me & others a favour and do a `if [[ "$(which spruce)X" == "X" ]]` to test if spruce is installed; and if not, show them where/how to get it?